### PR TITLE
tests/qa: trying $ distro mix

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

**Comment by yuriw** [comment]
here is how it looks on rados suite
http://pulpito.ceph.com/yuriw-2018-05-08_23:00:46-rados-mimic-distro-basic-ovh/

**Comment by yuriw** [comment]
@jdurgin here the latest configs http://pulpito.ceph.com/yuriw-2018-05-09_21:07:53-rados-mimic-distro-basic-ovh/
See rados/verify lines, not sure what wrong with it?

**Comment by yuriw** [comment]
From IRC:

(08:04:07 AM) yuriw: kefu_: http://pulpito.ceph.com/yuriw-2018-05-09_22:08:37-rados-mimic-distro-basic-smithi/ this first rados run on "$" implementation (e.g. mix of distro), anything jumps at you, specifically rhel related ?

(08:04:56 AM) ***kefu_ checking
(08:13:00 AM) kefu_: yuriw 2511364 is a known issue, http://pulpito.ceph.com/yuriw-2018-05-09_22:08:37-rados-mimic-distro-basic-smithi/2511313/ is a new one
(08:14:09 AM) kefu_: the latter is rhel 7.5, not sure if it's related. 
(08:14:17 AM) kefu_: i need to take a closer look tmr.

**Comment by yuriw** [comment]
rerun after removing `rhel` distro
http://pulpito.ceph.com/yuriw-2018-05-10_20:55:25-rgw-master-distro-basic-ovh/

File: qa/suites/rados/standalone/supported-random-distro$ [hunk 1]
```diff
@@ -0,0 +1 @@
+../basic/supported-random-distro$
```
Comment (jdurgin) [inline qa/suites/rados/standalone/supported-random-distro$]: need to move the workloads into a subdir, so they aren't all combined into one job

**Comment by yuriw** [comment]
run in rhel only as per @djgalloway rhel issue wasn't related to https://bugzilla.redhat.com/show_bug.cgi?id=1510658

so re-added rhel and testing again:
http://pulpito.ceph.com/yuriw-2018-05-11_21:05:52-rados-mimic-distro-basic-smithi/

**Comment by tchaikov** [comment]
reran the failed test of http://pulpito.ceph.com/yuriw-2018-05-09_22:08:37-rados-mimic-distro-basic-smithi/2511313/ at 
- http://pulpito.ceph.com/kchai-2018-05-14_06:47:12-rados-wip-kefu-testing-2018-05-12-1405-distro-basic-smithi/
- http://pulpito.ceph.com/kchai-2018-05-14_06:46:35-rados-wip-kefu-testing-2018-05-12-1405-distro-basic-mira/

both of them passed.

**Comment by yuriw** [comment]
@jdurgin ready to merge?

**Pull request merged**